manifests/q/qishibo/AnotherRedisDesktopManager/1.4.9/qishibo.AnotherRedisDesktopManager.locale.en-US.yaml

```diff
@@ -0,0 +1,26 @@
+# Created with YamlCreate.ps1 v2.0.0 using YAML parsing
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: qishibo.AnotherRedisDesktopManager
+PackageVersion: 1.4.9
+PackageLocale: en-US
+Publisher: qii404.me
```

Comment:
```suggestion
Publisher: Another
```

Reply:
aaaand another developer messed up his Publisher 🤦‍♂️ was ok in the previous version lmao
![image](https://user-images.githubusercontent.com/15158490/131563182-c0816d0b-a61a-421a-87b1-330af3ae82ef.png)